harness/tests/integration_cases/test_raft_paper.rs

```diff
@@ -647,23 +647,27 @@ fn test_follower_check_msg_append() {
     let ents = vec![empty_entry(1, 1), empty_entry(2, 2)];
     let mut tests = vec![
         // match with committed entries
-        (0, 0, 1, false, 0, 1),
-        (ents[0].term, ents[0].index, 1, false, 0, 1),
+        (0, 0, 1, 1, false, 0, 0),
```

Comment:
should still keep original `w_commit` field.

Reply:
I reorder `w_commit` before group of `reject`